### PR TITLE
Splash UI Tweaks

### PR DESCRIPTION
**Pull request by jumaallan**
Story: https://app.shortcut.com/smileid/story/xxx

## Summary

Splash UI Tweaks

## Known Issues

Any shortcomings in your work. This may include corner cases not correctly handled or issues related
to but not within the scope of your PR. Design compromises should be discussed here if they were not
already discussed above.

## Test Instructions

Concise test instructions on how to verify that your feature works as intended. This should include
changes to the development environment (if applicable) and all commands needed to run your work.

## Screenshot

If applicable (e.g. UI changes), add screenshots to help explain your work.

### CODE REVIEW

File: Example/SmileID/OnboardingScreen.swift
Comment (vanshg) [inline Example/SmileID/OnboardingScreen.swift:1]: Can you rename the file and composable to `WelcomeScreen` so that it matches Android?

File: Example/SmileID/OnboardingScreen.swift [hunk 1]
```diff
@@ -5,118 +5,148 @@ struct OnboardingScreen: View {
...
+                                                        .padding([.leading, .trailing], 20)
+                                        )
+                            }
+                        }
+                )
     }
 }
 
-private func updateSmileConfig(_ configJson: String) -> Bool {
+private func updateSmileConfig(_ configJson: String) -> (Bool, String) {
```
Comment (vanshg) [inline Example/SmileID/OnboardingScreen.swift:143]: This is redundant, we don't need to return a tuple here. We can instead just return `String?`. If not nil, it means the config was successfully read. Otherwise if it is nil, that inherently means there was an error

File: Example/SmileID/RootView.swift [hunk 2]
```diff
@@ -4,9 +4,12 @@ import SmileID
 struct RootView: View {
     // This is set by the SettingsView
     @AppStorage("smileConfig") private var configJson = (
-        UserDefaults.standard.string(forKey: "smileConfig") ?? ""
+            UserDefaults.standard.string(forKey: "smileConfig") ?? ""
     )
     private let jsonDecoder = JSONDecoder()
+    @State private var showSuccess = false
+    @State private var partnerId = ""
```
Comment (vanshg) [inline Example/SmileID/RootView.swift:11]: Would be much better if this was a nullable String, and not assigned to an empty string

**Review by vanshg — COMMENTED** [review]
- The manual config entry is missing the hint text
- The actual splash screen displays only the app text, but it is missing the logo
- Can you address all the lints
- There's a lot of reformatting in this PR. 2 asks: 1. can you revert those? and 2. can you update your spacing to indent 4 spaces instead of 8?

File: Example/SmileID/RootView.swift (hunk 2 above)
Comment (jumaallan) [reply Example/SmileID/RootView.swift]: Fixed

File: Example/SmileID/OnboardingScreen.swift (hunk 1 above)
Comment (jumaallan) [reply Example/SmileID/OnboardingScreen.swift]: Fixed

File: Example/SmileID/OnboardingScreen.swift
Comment (jumaallan) [reply Example/SmileID/OnboardingScreen.swift:1]: Done

File: Example/SmileID/WelcomeScreen.swift [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+                .overlay(
+                        Group {
+                            if showSuccess {
+                                Color.black.opacity(0.3)
+                                        .edgesIgnoringSafeArea(.all)
+                                        .overlay(
+                                                AlertView(
+                                                        icon: Image(systemName: "checkmark.circle.fill"),
+                                                        title: "Configuration Added",
+                                                        description: "Welcome partner \(String(describing: partnerId)), you can "
```
Comment (vanshg) [inline Example/SmileID/WelcomeScreen.swift:128]:
```suggestion
                                                        description: "Welcome Partner \(String(describing: partnerId)), you can "
```

**Comment by vanshg** [comment]
Still running into the issues I mentioned earlier:
- I'm getting a black screen instead of a splash screen when I launch the app
- The manual config entry is still missing the hint text - it's just a blank text box
- There's still a lot of reformatting in this PR. can you update your spacing to indent 4 spaces instead of 8?
- can you resolve the merge conflict?

**Review by vanshg — APPROVED** [review]
